radixdlt/src/main/java/com/radixdlt/consensus/VertexMetadata.java

```diff
@@ -46,35 +50,56 @@
 	@DsonOutput(Output.ALL)
 	private final long stateVersion;
 
+	@JsonProperty("isEndOfEpoch")

```

Comment:
We've been fairly consistently using snake_case on these.

Reply:
good catch

Reply:
Doesn't seem to be resolved?